MetaMask/metamask-extension
# fix: margin on asset chart min/max indicators

## **Description**

Design feedback that the min/max indicators on the asset chart should not be edge to edge like the chart itself.  They should have margin like the rest of the content on the page.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/27916?quickstart=1)

## **Related issues**

Fixes:

## **Manual testing steps**

1. Click on a token in the token list
2. Click various date range buttons
3. Verify min/max indicators on the chart are in right place
4. Verify min/max indicators keep 16px margin from the edges

## **Screenshots/Recordings**



### **Before**

![image](https://github.com/user-attachments/assets/1b319f18-277d-47a9-b3b2-90b854a58864)

### **After**

![image](https://github.com/user-attachments/assets/90a12c91-16cf-46b4-be91-c287d7d93bf8)

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
